cmd/skaffold/app/cmd/build.go

```diff
@@ -76,21 +76,23 @@ func runBuild(out io.Writer) error {
 
 	bRes, err := createRunnerAndBuildFunc(ctx, buildOut)
 
-	if err != nil {
```

Comment:
This error handling should not be removed, if there is a runner error for example we don't want to continue and ignore it.

Reply:
whoops don't know how this happened. fixed